app/models/agents/lifx_pulse_lights_agent.rb

```diff
@@ -0,0 +1,64 @@
+module Agents
+  class LifxPulseLightsAgent < Agent
+    include FormConfigurable
+    include Oauthable
+    valid_oauth_providers :'lifx'
+    
+    cannot_be_scheduled!
+    cannot_create_events!
+
+    description <<-MD
+      Pulses your your [LIFX lights](http://lifx.com/) between their current color and a color you specify.
+      
+      To be able to use this Agent you need to authenticate with LIFX in the [Services](/services) section first.
+
+      Read more about the [LIFX HTTP API](http://api.developer.lifx.com/) 
```

Comment:
I don't, having a link to the general API docs is fine as well. I am always a bit torn when it comes to documenting agent options that are also documented in external API documentation. It's either duplicating the docs (which might get outdated at one point), or pointing the user to the external docs which means opening another tab to read the docs there.